samples/bot-formatting-cards/csharp/README.md

```diff
@@ -125,6 +125,47 @@ Please find below demo manifest which is deployed on Microsoft Azure and you can
             }
             ]
         ```
+   
+   **Note: In Adaptive Card what we are defining (User details) should be exist in the same tenant where you are testing the app (teams login) etc...**
```

Comment:
**Note: In adaptive card, what we are defining (User details) should be exist in the same tenant where you are testing the app (teams' login) etc...**

Reply:
Fixed